autolab/Autolab
# Make assessment imports from file system have more clear error messages

## Description

Originally, when I was testing #1559, and copying over an assessment, I ended up getting a vague error:
<img width="1233" alt="Screen Shot 2022-07-12 at 21 23 31" src="https://user-images.githubusercontent.com/25730111/178895079-d991a4c6-75e6-472b-8a3f-ae5dbbe65626.png">

I eventually realized that the error was occurring because the folder name and its corresponding yml file for the assessment I was trying to import didn't have the same name. This PR aims to make this error easier to understand, as well as displaying an error if the folder name doesn't consist of accepted characters, which wasn't previously displayed. 

New error display:

<img width="1431" alt="Screen Shot 2022-07-13 at 20 46 28" src="https://user-images.githubusercontent.com/25730111/178895961-9cf12923-44b0-4050-9955-760579a26d52.png">


Very small changes, but I think when copying over assessments it might be easy to forget to follow these requirements which is why I think this is a good first step towards making the validation more transparent.

I also realized that if there were no acceptable existing assessments to import, the instructions on importing them would disappear. The page should now still show the instructions even if there are no assessments to import, since I think it's useful to see it regardless.

Before:
<img width="1008" alt="Screen Shot 2022-07-13 at 20 43 59" src="https://user-images.githubusercontent.com/25730111/178895666-bbb69c0f-1067-4b1a-92a8-340760699836.png">

After:
<img width="1211" alt="Screen Shot 2022-07-12 at 23 08 16" src="https://user-images.githubusercontent.com/25730111/178895885-010d4a4c-f50a-4e7a-a13a-5d953fb41e56.png">

## Motivation and Context



This PR is related to #1393 as it starts to give more detail in naming / yml standards for assessment importing, but doesn't address the core points of the issue. I will likely circle back and see if I can make some of the changes suggested in a later PR.

## How Has This Been Tested?



- Copy over an existing assessment, and make sure the folder name and yml file names don't match
- Try to install an assessment, and check that error message appears as above
- Make another copy of an existing assessment, and add a non-accepted character to the folder name (such as a space)
- see that another error message appears as above
- see that the steps to install an assessment from the file directory shows up without any assessments available to import

## Types of changes


## Checklist:
